app/helpers/fact_values_helper.rb

```diff
@@ -1,6 +1,6 @@
 module FactValuesHelper
   def fact_from(record)
-    _("%s ago") % time_ago_in_words(record.updated_at)
+    date_time_absolute(record.updated_at)
```

Comment:
Is this an explicit choice to go from relative to absolute?

Reply:
yes, the reasoning is the same as for reports/hosts page, the recommendation for tables is use the absolute date since the relative information for 20 lines usually ends up as "x days ago" which does not provide too much info

with facts it might vary a bit more but I think absolute date here makes more sense